webassembly/example.html

```diff
@@ -1,51 +1,168 @@
 <!DOCTYPE html>
-<html>
+<html data-theme="dark">
 
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>F3D Web</title>
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.0/css/bulma.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma-switch@2.0.4/dist/css/bulma-switch.min.css">
   <style>
-    html,
-    body {
-      height: 100%;
-      margin: 0;
-    }
-
     #main {
-      min-height: 60%;
+      min-height: 70vh;
+      border: 1px solid var(--bulma-primary);
+      border-radius: 0.5rem;
+      overflow: hidden;
     }
   </style>
 </head>
 
 <body>
-  <h1>F3D Web demo</h1>
-  <div id="main">
-    <canvas id="canvas"></canvas>
-  </div>
+  <section class="section">
+    <div class="container">
+      <h1 class="title">F3D</h1>
+      <p class="subtitle">
+        WebAssembly demo
```

Comment:
At this point its more than a demo to me. People will actually use this.

Reply:
the subtitle could be a quick description:

"View any 3D data" ? or something similar

Reply:
for reference: https://3dviewer.net/

Reply:
changed it